src/features/pastelDB/pastelDBLib.ts

```diff
@@ -120,27 +127,58 @@ export async function createTables(db: Database): Promise<void> {
   db.exec(createListunspent)
   db.exec(createTotalbalance)
   db.exec(createListaddresses)
+  db.exec(createPastelPriceTable)
   return
 }
 
 export function validateDataFromDB(
```

Comment:
This function starts to be too large. Could you please provide comment to it, and split it into multiple functions please?